packages/Heading/README.md

```diff
@@ -0,0 +1,68 @@
+## Heading
+
+The `<Heading>` component provides heading styles without className, which is already implemented in heading.js so can be used with the attributes to control the styling without importing from Heading.styles.js.If displayLevel exists, the heading will display <h> tag with displayLevel, otherwise it will display <h> tag with level.
```

Comment:
We want this to be human readable friendly and very simple. We don't need to discuss technical implementations here. Just What the component is.

Reply:
done